src/parent/pom.xml

```diff
@@ -514,6 +514,26 @@
           <generateGitPropertiesFilename>src/main/resources/git.properties</generateGitPropertiesFilename>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.7.5.201505241946</version>
+        <executions>
+          <execution>
+            <id>pre-unit-test</id>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>post-unit-test</id>
+            <phase>test</phase>
+            <goals>
+              <goal>report</goal>
```

Comment:
This goal generates the coverage reports. We can remove this execution, so the reports aren't always generated whenever tests are run. If reports are desired, simply add `jacoco:report` to the end of the mvn command. Eaxmple:
`mvn clean install jacoco:report`